ravenframework/Models/PostProcessors/ValidationBase.py

```diff
@@ -126,14 +126,14 @@ def _handleInput(self, paramInput):
     for child in paramInput.subparts:
       if child.getName() == 'pivotParameter':
         self.pivotParameter = child.value
-      elif child.getName() == 'Features':
-        self.features = child.value
-      elif child.getName() == 'Targets':
-        self.targets = child.value
+      elif child.getName() == 'prototypeOutputs':
+        self.prototypeOutputs = child.value
+      elif child.getName() == 'targetOutputs':
+        self.targetOutputs = child.value
```

Comment:
please update the manual to reflect these changes in validation related post-processors